earth_enterprise/src/common/compressor.h

```diff
@@ -187,7 +187,7 @@ struct ReadBuffer : public jpeg_source_mgr {
     ReadBuffer* src = (ReadBuffer*)cinfo->src;
```

Comment:
"boolean" may not necessarily be the same as the true C "bool" type, and hence return true may cast to something like (int)-1, where TRUE may be (int)1???   So I am wondering if we want to either keep the original TRUE/FALSE macros, or make sure we also use "bool".  I would have to dig a bit deeper to see how these were defined...

Reply:
addressed